src/topology/algebra/group.lean

```diff
@@ -2,23 +2,77 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro, Patrick Massot
-
-Theory of topological groups.
-
 -/
+
 import order.filter.pointwise
 import group_theory.quotient_group
 import topology.algebra.monoid
 import topology.homeomorph
 
+/-!
+# Theory of topological groups
+-/
```

Comment:
Do you mind expanding the module doc string a bit?

Reply:
Done.